en/animation/marionette/animation-graph.md

```diff
@@ -0,0 +1,35 @@
+# Animation Graph Assets
+
+The animation graph asset is used to store the entire animation flow data of the object. The flow of the animation is described by a state machine, currently an animation graph only supports one state machine.
+
+## Creating
+
+In the **Assets** panel click the **+** button on the top left, then select **Animation Graph**.
+
+![create-animation-graph](animation-graph/create-animation-graph.png)
+
+This creates an animation graph asset named `Animation Graph` by default.
+
+![animation-graph-asset](animation-graph/animation-graph-asset.png)
+
+## Editing
+
+Once the animation graph is created, you can edit it in the Animation Graph panel, please refer to the [Animation Graph Panel](animation-graph-panel.md) documentation for details.
```

Comment:
```suggestion
Once the animation graph is created, it can be edited it in the Animation Graph panel, please refer to the [Animation Graph Panel](animation-graph-panel.md) documentation for details.
```